packages/recomponents/src/components/r-modal/r-modal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <transition name="modal" @enter="enter" @leave="leave">
+    <transition name="r-modal"  @appear="appear" @enter="enter" @leave="leave">
```

Comment:
```suggestion
    <transition name="r-modal" @appear="appear" @enter="enter" @leave="leave">
```
Is that extra space typical?

Reply:
Oh, definitely no.